content/millennium/r4/scheduling/appointment.md

```diff
@@ -249,6 +249,17 @@ Create a new Appointment.
 
     POST /Appointment
 
+_Implementation Notes_
```

Comment:
We should also probably add some of the new business rules, thought I'm not sure how fine grained we want to be here (@MaxPhilips thoughts?).

Other potential rules we could add:
Participant status must be 'accepted'
Participant actor must be a patient reference (I believe this holds true for dstu2 but isn't on the implementation notes)

Reply:
do this:
```
* The modifier elements [implicitRules] and [modifierExtension] are not supported and will be rejected if present.
* `Appointment.status` must be set to `booked`.
* `Appointment.slot` must be a list containing a single reference to the Slot in which this appointment is being booked.
  * `Appointment.slot[0].reference` specifies an availability in the Scheduling system, which indicates details such as practitioner, location, and time.
* `Appointment.participant` must be a list containing a single participant.
  * `Appointment.participant.type` must not be set.
  * `Appointment.participant.actor` must be a reference to a Patient.
  * `Appointment.participant.status` must be set to `accepted`.
```

Reply:
updated in this commit: https://github.com/cerner/fhir.cerner.com/pull/311/commits/bca7479f502917dd24f775f8af2cb79f44092a44

@MaxPhilips Should dstu2 Implementation notes be formatted the same way with properties being indented?

Reply:
that would be nice

Reply:
updated formatting here: https://github.com/cerner/fhir.cerner.com/pull/311/commits/df416bf12a082a285576f4b1167824f34b08a323